test/Unit/Porter/Net/Caching/CachingConnectorTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+namespace ScriptFUSIONTest\Unit\Porter\Net\Caching;
+
+use Mockery\Adapter\Phpunit\MockeryPHPUnitIntegration;
+use ScriptFUSION\Porter\Cache\MemoryCache;
+
+final class CachingConnectorTest extends \PHPUnit_Framework_TestCase
+{
+    use MockeryPHPUnitIntegration;
+
+    public function test()
+    {
+        $memoryCache = new MemoryCache();
```

Comment:
For this to quality as a unit test it can only instantiate the system under test (SUT). Either mock the cache or move this to the integration test class.